src/Files.App/Views/Settings/TagsPage.xaml

```diff
@@ -166,15 +157,15 @@
 
 						<ListView
 							x:Name="TagsList"
+							Grid.Row="1"
 							MaxHeight="500"
 							Padding="12"
-							HorizontalAlignment="Stretch"
 							AllowDrop="True"
 							CanReorderItems="True"
 							IsItemClickEnabled="True"
 							ItemsSource="{x:Bind ViewModel.Tags, Mode=TwoWay}"
 							ReorderMode="Enabled"
-							SelectionMode="None">
+							SelectionMode="Single">
```

Comment:
Why was this changed?

Reply:
Fixed

Reply:
Did you have a reason to change this? I'm not against the change but I'd like to know the reasoning.

Reply:
Fixed